assembl/models/post.py

```diff
@@ -57,6 +57,8 @@ class PublicationStates(DeclEnum):
     MODERATED_TEXT_NEVER_AVAILABLE = "MODERATED_TEXT_NEVER_AVAILABLE", ""
     DELETED_BY_USER = "DELETED_BY_USER", ""
     DELETED_BY_ADMIN = "DELETED_BY_ADMIN", ""
+    WIDGET_SCOPED = "WIDGET_SCOPED", ""
```

Comment:
There is a TODO note in generic.py:
# TODO: Refactor hidden into PublicationStates.WIDGET_SCOPED
If the type is left out, the TODO is harder to understand; I think the TODO is meaningful. Of course the TODO would be better expressed as a task in jira.